src/section.js

```diff
@@ -88,16 +88,9 @@ class Section {
 
 		this.load(_request).
 			then(function(contents){
-				var serializer;
-				var Serializer;
-
-				if (typeof XMLSerializer === "undefined") {
-					Serializer = require("xmldom").XMLSerializer;
-				} else {
-					Serializer = XMLSerializer;
```

Comment:
HI @fchasen, Sorry for delayed response, have done changes as per your review comments and added a condition to use "xmldom" for IE. Have tested the changes works fine.
Requesting your review.